.pylintrc

```diff
@@ -22,8 +21,17 @@ disable=
   too-many-nested-blocks,
   too-many-public-methods,
   too-many-statements,
-  too-public-methods,
   wrong-import-order,
+  import-error,
+  consider-using-f-string,
+  use-implicit-booleaness-not-comparison,
+  unspecified-encoding,
+  no-name-in-module,
+  unnecessary-dunder-call,
+  use-list-literal,
+  use-dict-literal,
+  deprecated-module,
```

Comment:
`import-error`: pylint is no longer installed with all project dependencies, this is covered by mypy + runtime tests.
`consider-using-f-string`, `use-list-literal`, `use-dict-literal`: Covered by [pyupgrade](https://github.com/aiven/karapace/pull/530).
`unspecified-encoding`: Covered by pyupgrade, but with different resolution.
`use-implicit-booleaness-not-comparison`: I think this is overly opinionated. `foo == []` is fine and doesn't need to be outlawed.